src/App.js

```diff
@@ -1,26 +1,114 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react'
+import "bulma/css/bulma.css";
 import './App.css';
+import foods from "./foods.json";
+import FoodList from './components/FoodList';
+import AddFoods from './components/AddFoods';
+import TodaysFoods from './components/TodaysFoods';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+export default class App extends Component {
+
+  state = {
+    foodList: foods,
+    filteredFoods: foods,
+    todaysFoods: [],
+    showAddForm: false
+  }
+
+  handleShowAddForm = () => {
+    this.setState({
+      showAddForm: true
+    })
+  }
+
+  handleAddFood = (e) => {
+    e.preventDefault();
+
+    let newFood = {
+      name: e.target.name.value,
+      calories: e.target.calories.value,
+      image: e.target.image.value,
+    }
+
+    this.setState({
+      foodList: [...this.state.foodList, newFood],
+      filteredFoods: [...this.state.filteredFoods, newFood],
+      showAddForm: false
+    })
+  }
+
+  handleSearch = (e) => {
+    let search = e.target.value;
+    const { foodList } = this.state;
```

Comment:
Same as here, you could have do a destructure of the state in the previous function.